app/assets/stylesheets/header.css.scss

```diff
@@ -1,7 +1,22 @@
+@import "colors";
+
 .lineup{
   display: inline-block;
 }
 
+#header{
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 55px;
+  z-index: 101;
+  background-color: $tertiary-background-color;
+}
+
+.header-wrapper{
```

Comment:
unused classes should be removed... both in css and html
if you start creating classes because "maybe I'll use it later" your code is gonna get very messy soon